frontend/src/components/NavBarMenu.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+import React, { useState } from "react";
+
+type MenuItem = {
+  label: string;
+  url: string;
+};
+
+const menuItems: MenuItem[] = [
+  {
+    label: "Books",
+    url: "/books",
+  },
+  {
+    label: "Characters",
+    url: "/characters",
+  },
+  {
+    label: "Movies",
+    url: "/movies",
+  },
```

Comment:
Now that we also have spells and potions, it would be nice to add those, too. Also make sure to merge to incoming changes from the `new-website` branch.